l10n/app_zh.arb

```
@@ -1651,6 +1651,13 @@
   "versionPageReleaseNotes": "发布说明",
   "versionPageUpdateAvailable": "v{version} 已发布",
   "appReleaseUpdateDialogBody": "发现新版本应用。",
```

RuboCop (1.88.2)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)